easybuild/easyconfigs/m/MM-align/MM-align-1.0-goolf-1.4.10-20130815.eb

```diff
@@ -0,0 +1,35 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'CmdCp'
+
+name = 'MM-align'
+version = '1.0' # no version info provided. I decide it's 1.0
```

Comment:
@pescobar: I would replace this with `20130815` rather than picking a random version number (and drop the `versionsuffix`)